.changeset/spotty-dogs-wonder.md

```diff
@@ -0,0 +1,5 @@
+---
+'@graphcms/rich-text-types': minor
+---
+
+Update EmbedProps type
```

Comment:
Maybe we could write something more descriptive, such as:
`Add isInline to EmbedProps type`